service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapperTest.java

```diff
@@ -284,7 +288,8 @@ public void When_MappingObservationWithInvalidParticipantResourceType_Expect_Exc
             Arguments.of(INPUT_JSON_WITH_NO_COMPONENT, OUTPUT_XML_WITH_NO_COMPONENT),
             Arguments.of(INPUT_JSON_WITH_PARTICIPANT, OUTPUT_XML_WITH_PARTICIPANT),
             Arguments.of(INPUT_JSON_WITH_EFFECTIVE_PERIOD_NO_START, OUTPUT_XML_WITH_EFFECTIVE_TIME_UNK_LOW),
-            Arguments.of(INPUT_JSON_WITH_EFFECTIVE_PERIOD_BLANK, OUTPUT_XML_WITH_EFFECTIVE_TIME_UNK)
+            Arguments.of(INPUT_JSON_WITH_EFFECTIVE_PERIOD_BLANK, OUTPUT_XML_WITH_EFFECTIVE_TIME_UNK),
+            Arguments.of(INPUT_JSON_WITH_REFERENCE_RANGE_NO_VALUE_QUANTITY, OUTPUT_XML_WITH_REFERENCE_RANGE_NO_VALUE_QUANTITY)
         );
```

Comment:
Is it possible for there to only be a low or only a high value? Would it be worth adding tests for these?

Reply:
Wouldn't do any harm, thanks